resources/styles/components/task-step/all-steps.less

```diff
@@ -6,7 +6,7 @@
     width: 960px;
     height: 500px;
```

Comment:
Not really related to this pr, but you might update the 960 to your new variable (that I can't remember the name of) :)